app/src/main/kotlin/io/imhungry/notifications/NotificationHelper.kt

```diff
@@ -0,0 +1,97 @@
+package io.imhungry.notifications
+
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.app.PendingIntent
+import android.content.Context
+import android.os.Build
+import androidx.core.app.NotificationCompat
+import androidx.core.app.NotificationManagerCompat
+import io.imhungry.R
+
+class NotificationHelper(private val context: Context) {
+
+    /**
+     *  Unfortunately, NotificationCompat does actually have an ENUM behind the scenes.
+     *  This simplifies the function arguments in this class.
+     */
+    enum class Priority constructor(internal val value: Int) {
+        MIN(NotificationCompat.PRIORITY_MIN),
+        LOW(NotificationCompat.PRIORITY_LOW),
+        DEFAULT(NotificationCompat.PRIORITY_DEFAULT),
+        HIGH(NotificationCompat.PRIORITY_HIGH),
+        MAX(NotificationCompat.PRIORITY_MAX)
+    }
+
+    /**
+     * Need to yell at the user right away? This is your function!
+     * Just pass what you want to say. That's it.
+     */
+    fun sendNotificationNow(
+        title: String,
+        message: String,
+        intent: PendingIntent?,
+        priority: Priority,
+        notificationID: Int
+    ) {
+        scheduleNotification(
+            getStarterNotificationBuilder(title, message, priority).setContentIntent(intent),
+            notificationID
+        )
+    }
+
+    /**
+     * All you have to do is create the notification, this will take care of the rest.
+     */
+    fun scheduleNotification(
+        notificationBuilder: NotificationCompat.Builder, notificationID: Int
```

Comment:
```suggestion
        notificationBuilder: NotificationCompat.Builder,
        notificationID: Int
```
These arguments should be on separate lines.

Reply:
Auto format doesn't grab it